src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,50 @@
+.calendar {
```

Comment:
you have forgotten to display a calendar in the middle of the screen

Reply:
Oh, really, thanks!